cookies/views.py

```diff
@@ -1179,3 +1180,41 @@ def resource_content(request, resource_id):
                 target += '?accessToken=' + auth_token
         return HttpResponseRedirect(target)
     return HttpResponse('Nope')
+
+def export_coauthor_data(request):
+    context = RequestContext(request, {})
+
+    graph = nx.Graph()
+
+    parent_id = request.GET.get('parent_collection', None)
```

Comment:
It looks like this entire view depends on having a specified collection; is that right? If so, why not make the collection ID a parameter in the URL itself, rather than part of the GET payload?